server/plugin/command.go

```diff
@@ -367,7 +367,17 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.BotUserID,
+		Message:   msg,
+	}
+
+	if _, err := p.API.CreatePost(post); err != nil {
+		p.API.LogWarn("Error webhook post", "post", post, "error", err.Error())
```

Comment:
Copy past error?

Reply:
@hanzei  sorry i didn't got this

Reply:
Why does the log message say `Error webhook post`?